docs/developer/conventions/code/qa.rst

```diff
@@ -79,7 +79,7 @@ Error Prone
 
 The `Error Prone <https://errorprone.info/>`_ checker runs a compiler plugin.
 
-In order to activate the Error Prone checks, use the "-Perrorprone" for JDK 11 builds, or "-Perrorprone8" for JDK 8 builds.
+In order to activate the Error Prone checks, use the "-Perrorprone". For JDK 8 the profile "errorprone-jdk8-fix" is automatically added.
```

Comment:
I already asked once not to call that profile "fix", do not change the documentation and revert the name change in the pom please.